launch/gazebo2rviz.launch

```diff
@@ -1,4 +1,10 @@
 <launch>
```

Comment:
Thanks, I see you've found out that Github is clever enough to solve this kind of things =) I just split the PR into two pieces, because I wasn't sure if you're interested in merging the model-URI handler.